vernac/g_vernac.mlg

```diff
@@ -525,9 +526,16 @@ GRAMMAR EXTEND Gram
     [ [ id = identref; c=constructor_type -> { c id } ] ]
   ;
   of_type:
-    [ [   ":>" -> { BackInstance }
-        | ":"; ">" -> { BackInstance }
-        | ":" -> { NoInstance } ] ]
+    [ [   ":>" -> { true }
```

Comment:
This should be some Coercion | NotCoercion type instead of bool

Reply:
You're right, let's take the opportunity to improve that.